pkg/model/stack_expander_test.go

```diff
@@ -81,15 +87,14 @@ services:
     myservice:
         build:
             context: vote
-        args:
-            - $CUSTOM_ENV
-            - CUSTOM2_ENV=$CUSTOM_ENV
+            args:
+                - CUSTOM2_ENV=$CUSTOM_ENV
         ports:
             - 8080:8080
         environment:
             FLASK_ENV: development
             CUSTOM_ENV: $CUSTOM_ENV
-            $CUSTOM2_ENV:
+            CUSTOM2_ENV:
```

Comment:
Same about the other (previous) case. Is it something we are changing?

Reply:
We do translate it, and the initial implementation of the stack expander was taking into account this logic, but this does not comply with the compose spec, which what it does is that it expands the value, not making the key-value trick. What should we do in this case? should i recover then the logic regarding the expansion of this case into key-value? 

cc: @jLopezbarb @pchico83

Reply:
@teresaromero on which cli version was that logic added? I think that behavior is wrong, and the sooner we delete it, the better. I don't think people are using it that way unless it was implemented for a user feature request

Reply:
if we think it is not being used, agree with deleting it